processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -115,7 +115,56 @@ public BitmapFactory getBitmapFactory()
   public BitmapIndex getBitmapIndex(String dimension)
```

Comment:
we probably also need to modify  getBitmapIndex(String dimension, String value) below in this class so as to return null when column does not have bitmap index ? 
would also be nice to have the same logic at one place and use it in both the methods.